lib/audit.js

```diff
@@ -30,8 +33,8 @@ module.exports = async function auditCore (core, update) {
     if ((node.index & 1) === 0) continue
 
     const [left, right] = flat.children(node.index)
-    const leftNode = getNode(left)
-    const rightNode = getNode(right)
+    const leftNode = tree.get(left)
+    const rightNode = tree.get(right)
```

Comment:
important bug fix on main here (why its conflicting) lets merge main in (unrelated to your PR)